projects/robots/pal_robotics/tiago_titanium/worlds/tiago_titanium.wbt

```diff
@@ -0,0 +1,392 @@
+#VRML_SIM R2019b utf8
+WorldInfo {
+  info [
+    "This simulation shows a Tiago Titanium making hello with the sign "Peace" with these fingers in an industrial environment. It can also be controlled using the arrows on the keyboard."
```

Comment:
```suggestion
    "This simulation shows a Tiago Titanium making hello with the sign 'Peace' with these fingers in an industrial environment. It can also be controlled using the arrows on the keyboard."
```